LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/Payments/Payrun.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Common.AppConstants.Enums;
+using LBH.AdultSocialCare.Api.V1.AppConstants.Enums;
+
+namespace LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.Payments
+{
+    public class Payrun
+    {
+        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public Guid Id { get; set; }
+
+        public PayrunType Type { get; set; }
+        public PayrunStatus Status { get; set; }
+
+        public bool Paid { get; set; }
+        public bool Held { get; set; }
+
+        public DateTimeOffset PaidUpToDate { get; set; }
```

Comment:
Paid and Held bools can be done in payrun status I believe. PaidUpToDate not sure it's required here. Can be on package though

Reply:
Date created, created by ...

Reply:
Paid and Held replaced with decimal, DateCreated etc. added

@brkzkn, please review the rest